archeo/functions.php

```diff
@@ -60,5 +60,43 @@ function archeo_styles() {
 
 add_action( 'wp_enqueue_scripts', 'archeo_styles' );
 
-// Add block patterns
-require get_template_directory() . '/inc/block-patterns.php';
+/**
+ * Registers block patterns and categories.
+ *
+ * @since Archeo 1.0
+ *
+ * @return void
+ */
+function archeo_register_block_patterns() {
```

Comment:
Nit but would this be a more accurate name for this function?

```suggestion
function archeo_register_block_pattern_categories() {
```